projects/packages/publicize/src/class-publicize.php

```diff
@@ -613,6 +614,7 @@ public function test_connection( $service_name, $connection ) {
 			'user_can_refresh' => $user_can_refresh,
 			'refresh_text'     => $refresh_text,
 			'refresh_url'      => $refresh_url,
+			'code'             => $connection_error_code,
```

Comment:
I'm not sure but I'm not seeing the notices as the error code isn't coming through for some reason - this is with Jetpack Social.

I wondered if this should be 
```suggestion
			'error_code'             => $connection_error_code,
```

But it doesn't seem to make a difference!